plugins/dashboard/explorer_routes.go

```diff
@@ -250,10 +250,15 @@ func findAddress(strAddress string) (*ExplorerAddress, error) {
 		// how much pending mana the output has?
 		pendingMana, _ := messagelayer.PendingManaOnOutput(output.ID())
 
+		jsonMetadata := jsonmodels.NewOutputMetadata(metaData)
+		messagelayer.Tangle().LedgerState.ConfirmedConsumer(output.ID()).Consume(func(consumer *ledgerstate.Consumer) {
```

Comment:
Same here, could be that there is no `ConfirmedConsumer` causing a nil pointer panic.